stylesheets/_touch-friendly-links.scss

```diff
@@ -0,0 +1,10 @@
+// This mixin will expand the clickable area for a link on mobile and tablet to make it easier to click on a touch
+// device.
+
+@mixin touch-friendly-links($padding: 3px) {
+  a {
+    padding: $padding;
+    margin: -1 * $padding;
+    outline: none;
```

Comment:
Users with impaired vision might want to increase contrast (in addition to perhaps zooming in, or overriding the font) – conversely, someone with dyslexia might choose to _reduce_ the contrast by using e.g. a peach background which reduces the [visual stress](http://www.bdadyslexia.org.uk/dyslexic/eyes-and-dyslexia#Visual Stress).